src/main/java/reposense/RepoSense.java

```diff
@@ -124,6 +124,7 @@ public static void main(String[] args) {
             // help message was printed by the ArgumentParser; it is safe to exit.
         }
 
+        LogsManager.moveLogFileToOutputFolder();
```

Comment:
I believe that the warning not caught can possibly be fixed by just setting the location of the logs earlier before all the warnings, rather than moving things around.

Reply:
When bringing up the issue, I did think of this but it's a bit of a strange problem because this method would immediately require the out folder argument always being the first argument that we read so that we can set the logs output folder. Though not really a big issue now, some scenarios this might be an issue is for example in your setup wizard where if you  want to include an output folder prompt together with preemptive argument checking, you would need to provide the output folder prompt as a first option otherwise you can't log anything.

I guess the main consideration is whether you believe there's anything worth logging between the start of a reposense run and the earliest moment the output folder location can be known. I think it's a relatively trivial change to implement to be more flexible which is why I suggested it.

Reply:
In this case, I do believe it will be better since it covers all grounds rather than rely on the positioning of the output from parse.

Reply:
Yup I agree, for instance, a log could be required before `ArgsParser::parse` is even called, in that case having the logs output in a temp directory first would be ideal.